pyuvdata/uvcal.py

```diff
@@ -221,8 +221,6 @@ def __init__(self):
                 'if cal_type is delay, shape is (Nspws, 1, Ntimes, Njones), '
                 'type = float.')
         self._total_quality_array = uvp.UVParameter('total_quality_array', description=desc,
-                                                    form=('Nspws', 'Nfreqs',
```

Comment:
These 2 lines should not be removed, they represent the default (just like we have for the quality_array). They can then get updated in the set functions.